src/tinyformat.h

```diff
@@ -490,7 +490,7 @@ namespace detail {
 
 // Type-opaque holder for an argument to format(), with associated actions on
 // the type held as explicit function pointers.  This allows FormatArg's for
-// each argument to be allocated as a homogenous array inside FormatList
+// each argument to be allocated as a homogeneous array inside FormatList
```

Comment:
NACK. Should go upstream

Reply:
Removed, thanks!

Reply:
Fun fact - tinyformat has this fixed upstream:

https://github.com/c42f/tinyformat/blob/4d03ee3d04c33c158bd261598ebde265885a6d9f/tinyformat.h#L496